app/views/users/registrations/edit.html.slim

```diff
@@ -51,6 +51,14 @@
               autocomplete: 'new-password',
               class: 'form-control'
 
+    .form-group.field-element
```

Comment:
Shall we add a small hint (see example in line 45) about the key's usage and where to get it (https://platform.openai.com/api-keys)?

Reply:
Thats a good idea. will add that now. :)